cookbook/frontend/bower.rst

```diff
@@ -25,8 +25,8 @@ it's installed correctly.
 .. tip::
 
     If you don't want to have NodeJS on your computer, you can also use
-    BowerPHP_ (an unofficial PHP port of Bower). Beware that this is still in
-    an alpha state. If you're using BowerPHP, use ``bowerphp`` instead of
+    BowerPHP_ (an unofficial PHP port of Bower). Be aware that this is currently
```

Comment:
Doesn't really make a difference to me - the version being the only important bit!